src/ddmd/glue.d

```diff
@@ -977,11 +977,10 @@ void FuncDeclaration_toObjFile(FuncDeclaration fd, bool multiobj)
             break;
     }
 
-    IRState irs = IRState(m, fd);
     Dsymbols deferToObj;                   // write these to OBJ file later
-    irs.deferToObj = &deferToObj;
+    Array!(elem*) varsInScope;
     Label*[void*] labels = null;
```

Comment:
Wow, we are actually using D's AAs in dmd. Nice :)

Reply:
:+1: